.gitpod.Dockerfile

```diff
@@ -0,0 +1,95 @@
+#
+# Dockerfile for jupyter/accessibility development
+#
+# Usage:
+# -------
+#
+# From the root of the project:
+# docker build --rm -f "./tools/Dockerfile" -t <build-tag> "."
+#
+# To run the image:
+# docker run --rm -it <build-tag>
+# This image is based on: Ubuntu 20.04 (focal)
+# https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
+# OS/ARCH: linux/amd64
+
+FROM gitpod/workspace-base:latest
+
+ARG MAMBAFORGE_VERSION="4.10.0-0"
+ARG CONDA_ENV=a11y-tests
+ARG DEBIAN_FRONTEND=noninteractive
+
+# ---- Configure environment ----
+# base HOME for all things in gitpod
+ENV GITPOD_HOME=/home/gitpod \
+    WORKSPACE=/workspace/a11y
+
+ENV CONDA_DIR="${GITPOD_HOME}/mambaforge3" \
+    SHELL=/bin/bash
+ENV PATH=${CONDA_DIR}/bin:$PATH
+
+# -----------------------------------------------------------------------------
+# ---- Creating as root - note: make sure to change to gitpod in the end ----
+USER root
+
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install -yq --no-install-recommends \
+    # software-properties-common \
+    # locales \
+    # wget \
+    build-essential \
+    ca-certificates \
+    curl \
+    dirmngr \
+    make && \
+    # this needs to be done after installing dirmngr
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-key C99B11DEB97541F0 && \
+    apt-add-repository https://cli.github.com/packages && \
+    apt-get install -yq --no-install-recommends \
+    gh && \
+    locale-gen en_US.UTF-8 && \
+    # clean apt cache
+    apt-get clean && \
+    rm -rf /var/cache/apt/* &&\
+    rm -rf /var/lib/apt/lists/* &&\
+    rm -rf /tmp/*
+
+# Allows this Dockerfile to activate conda environments
+SHELL ["/bin/bash", "--login", "-o", "pipefail", "-c"]
+
+# -----------------------------------------------------------------------------
+# ---- Installing mamba  ----
+USER gitpod
```

Comment:
I switch the user here because if the user is root when mamba is installed then it makes it hard or impossible for a developer once they're in a Gitpod workspace to create new environments because `root` ends up being the owner of all of the conda-related directories.

Reply:
So I just tried out what would happen on Github if I saved things to the filesystem between sessions with the same user and I observed two seemingly contradictory results.

1: Putting a file in /home/gitpod does not persist across sessions.

1. I opened a Gitpod workspace
2. `cd /home/gitpod`
3. `touch helloworld`
4. Shut down workspace
5. Re-open same workspace
6. `ls /home/gitpod`
-> No helloworld file

2: Putting a file in /workspace/accessibility does persist across sessions.

1. I opened a Gitpod workspace
2. `cd /workspace/accessibility`
3. `touch helloworld`
4. Shut down workspace
5. Re-open same workspace
6. `ls /workspace/accessibility`
-> helloworld file is there

Reply:
> On the other hand, are Gitpod instances reused between end users? 
No, they are not - each user will have their own instance. You can share it with other folks through a link but that is the only way

By default, the only directory that is persisted in an instance (note this instance is only kept for 15 days) is whatever is in `workspace/....` which is why this is the default working directory. 

`home` will only have by default whatever is in `home` at container build 😉  which is why I had chosen as the default location for the browsers - and also found a couple of issues and posts mentioning that this helps prevent issues with access or not finding the browsers in containers contexts